tests/ArrayToXmlTest.php

```diff
@@ -1,424 +1,314 @@
-    {
-        $this->expectException('DOMException');
-
-        ArrayToXml::convert(['one', 'two', 'three']);
-    }
-
-    /** @test */
-    public function it_throws_an_exception_when_converting_an_array_with_invalid_characters_key_names()
-    {
-        $this->expectException('DOMException');
-
-        echo ArrayToXml::convert(['tom & jerry' => 'cartoon characters'], '', false);
-    }
-
-    /** @test */
-    public function it_will_raise_an_exception_when_spaces_should_not_be_replaced_and_a_key_contains_a_space()
-    {
-        $this->expectException('DOMException');
-
-        ArrayToXml::convert($this->testArray, '', false);
-    }
-
-    /** @test */
-    public function it_can_handle_values_as_basic_collection()
-    {
-        $this->assertMatchesXmlSnapshot(ArrayToXml::convert([
-            'user' => ['one', 'two', 'three'],
-        ]));
-    }
-
-    /** @test */
-    public function it_can_handle_zero_values_in_beginning_of_basic_collection()
-    {
-        $this->assertMatchesXmlSnapshot(ArrayToXml::convert([
-            'user' => ['0', '1', '0'],
-        ]));
-    }
-
-    /** @test */
-    public function it_can_handle_values_with_namespace_as_basic_collection()
-    {
-        $root = [
-            'rootElementName' => 'root',
-            '_attributes' => [
-                'xmlns:ns1' => 'https://github.com/spatie/array-to-xml',
+            [
+                'name' => 'twee',
+                'age' => 12,
             ],
-        ];
-
-        $this->assertMatchesXmlSnapshot(ArrayToXml::convert([
-            'ns1:user' => ['one', 'two', 'three'],
-        ], $root));
-    }
-
-    /** @test */
-    public function it_accepts_an_xml_encoding_type()
-    {
-        $this->assertMatchesXmlSnapshot(ArrayToXml::convert([], '', false, 'UTF-8'));
-    }
-
-    /** @test */
-    public function it_accepts_an_xml_version()
-    {
-        $this->assertMatchesSnapshot(ArrayToXml::convert([], '', false, null, '1.1'));
-    }
-
-    /** @test */
-    public function it_accepts_an_xml_standalone_value()
-    {
-        $this->assertMatchesSnapshot(ArrayToXml::convert([], '', false, null, '1.0', [], false));
-    }
-
-    /** @test */
-    public function it_can_handle_values_as_collection()
-    {
-        $this->assertMatchesXmlSnapshot(ArrayToXml::convert([
-            'user' => [
-                [
+        ],
+    ]));
+});
+
+test('It can handle values with special characters', function () {
+    assertMatchesXmlSnapshot(ArrayToXml::convert(['name' => 'this & that']));
+});
+
+test('It can handle values with special control characters', function () {
+    assertMatchesXmlSnapshot(ArrayToXml::convert(['name' => "i want tothis and \x03 that"]));
+});
+
+test('It can group by values when values are in a numeric array', function () {
+    assertMatchesXmlSnapshot(ArrayToXml::convert(['user' => ['foo', 'bar']]));
+});
+
+test('It can convert attributes to xml', function () {
```

Comment:
Could you use  the `it` function instead of `test` where the test name begins with "it"?

Reply:
👍🏻 changed